frontend/src/models/associations/election.ts

```diff
@@ -2,23 +2,33 @@ import { User } from "../user";
 import {Association} from "./association";
 
 export interface Choice {
+    id: number;
+    name: string;
+    election: Election | number;
+}
+
+export interface Result {
     id: string;
-    election: Election;
+    result: string;
+    numberOfVoters:string;
+    numberOfRegistered:string;
 }
 
 export interface Ballot {
-    id: string;
+    id?: string;
     election: Election;
-    choices: Choice[];
+    choices: Choice[] | number[];
```

Comment:
Why do you use type union here? We should be confident with the data types we have in the models

Reply:
Hi, I had a problem because what I receive from python and what I send are different things...
For example, here in `Ballot`, I receive `Choice`s, but when someone votes, he sends only the ids of the choice, hence the `number[]`.
Similarly, in `Election`, I receive `Choice`s, but when I create an election, I have to send an `Object` with only a `name` field, hence the `Object[]`.
Again for `registeredVoters` in `Election`, I send only their ids as a `string[]`

(I removed one which was useless